Scripts/combo_dict.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Are we going to be creating a lot of these json files?  I suspect so, in which case we should have a dedicated directory for them.  mu2e_trig_config puts json files in the data subdirectory, I suggest that.